src/Diagnostics.DataProviders/DataProviderConfigurations/KustoDataProviderConfiguration.cs

```diff
@@ -53,6 +57,25 @@ public class KustoDataProviderConfiguration : IDataProviderConfiguration
         /// </summary>
         public ConcurrentDictionary<string, string> RegionSpecificClusterNameCollection { get; set; }
 
+        public string CloudDomain
+        {
+            get
+            {
+                if (AADKustoResource.Contains("windows.net"))
+                {
+                    return AzureCloud;
+                }
+                else if (AADKustoResource.Contains("chinacloudapi.cn"))
+                {
+                    return AzureChinaCloud;
+                }
+                else
```

Comment:
>else [](start = 16, length = 4)

Do you want to check the check for Blackforest (.cloudapi.de) or is black forest not in scope yet ?

Reply:
Correct, not within the scope